### PR TITLE
Add docker instructions

### PR DESCRIPTION
**Pull request by leoank**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,3 +67,27 @@ Prod:
...
+
+## Docker
+
+### Build docker image
+
+```bash
+docker build -f Dockerfile -t intermine/wizard:latest .
+```
+
+### Run wizard in a docker container
```
Comment (yochannah) [inline README.md:79]: Does this command have other dependencies?

Comment (yochannah) [reply README.md:79]: (e.g. containers running on some other port, etc.)

Comment (leoank) [reply README.md:79]: Nope. Just docker

Comment (leoank) [reply README.md:79]: And docker-compose for using docker-compose commands

Comment (yochannah) [reply README.md:79]: Hmm - when I run ```docker run --rm -p 9992:8080 intermine/wizard:latest ``` it tries to contact 127.0.0.1:9991 (and errors out)

Comment (leoank) [reply README.md:79]: you can use docker-compose file to start wizard with all InterMine cloud dependencies in docker containers.

Comment (yochannah) [reply README.md:79]: right! But in that case we should explain that _this_ command expects other things to be running at certain locations, and to use docker-compose if you want to start them all at once, I think?

Comment (leoank) [reply README.md:79]: Added a note about this in the README

**Review by yochannah — APPROVED** [review]

**Pull request merged**